zeeguu/api/endpoints/user.py

```diff
@@ -131,23 +131,20 @@ def user_settings():
         user.set_native_language(submitted_native_language_code)
 
     # deprecating the larned_language_code
```

Comment:
I'm not sure why this message is here. 

Could it be that it refers to the choosing `learned_langauge_code` or `learned_language`? We should look in the web, see which is the one used, and only keep that. The same above with `native_language_code` and `native_language`.